core/tests/protractor_utils/ContributorDashboardTranslateTextTab.js

```diff
@@ -88,6 +86,9 @@ var ContributorDashboardTranslateTextTab = function() {
   };
 
   this.expectSelectedLanguageToBe = async function(language) {
+    await waitFor.visibilityOf(
+      selectedLanguageElement,
+      'Selected language element taking too long to show up.');
     expect(await selectedLanguageElement.getText()).toMatch(language);
```

Comment:
Please add an action.getText function and use it here! (It's out of the context of this PR but I think this will help us avoid resisting for more fixes.)